fdbclient/include/fdbclient/EncryptKeyProxyInterface.h

```diff
@@ -102,18 +102,17 @@ struct EKPBaseCipherDetails {
 	constexpr static FileIdentifier file_identifier = 2149615;
 	int64_t encryptDomainId;
 	uint64_t baseCipherId;
-	StringRef baseCipherKey;
+	Standalone<StringRef> baseCipherKey;
```

Comment:
We need to run perf, but given the path gets invoked on KeyMiss only and we incur penalty of reaching out to external KMS and the payload here is roughly 60sh byte per tenant; expectation is it might be fine. 
Will run perf and if there are issues, will revert it back to use in-built area if needed.